pages/posts.js

```diff
@@ -0,0 +1,60 @@
+import Timeline from "@/components/TimeLine";
+import PostsComponent from "@/components/PostsComponent";
+import SmallScreenToggle from "@/components/SmallScreenToggle";
+
+export default function posts() {
+    const response = {
+        2021: [
```

Comment:
you wanna keep the mock data?

Reply:
everything else seems good

Reply:
yee for now lets keep.. will remove later